### DISCUSSION TITLE
ddt4all crashes when trying to load screens

### DISCUSSION THREAD

**Discussion by neilnutt** [cedricp/ddt4all#941] [Q&A]
I am able to load ddt4all and complete a scan. ddt4all unexpectedly quits a short moment after I click through to see a screen (screen never shows).  It doesn't matter which ecu I try to access, they all get the same unexpected quit with no warning raised.

Screenshot showing last moment before crash
![image](https://github.com/cedricp/ddt4all/assets/7341783/f029c36f-4308-44c4-a7ca-fda9f3d4428d)

Log file
[elm_ddt.log](https://github.com/cedricp/ddt4all/files/13232192/elm_ddt.log)

**Comment by AndyIreland** [comment]
what happens if you give this version a try - choose 'all cars' it has a built in database 
[https://www.drive2.ru/b/498093336985338243/](url)

**Comment by neilnutt** [comment]
Yes. The copy on the Russian site works. Doesn't resolve the issue for the current release, but fixes it for me.

**Comment by Furtif** [comment]
🧐

**Comment by KarelSvo** [comment]
> I am able to load ddt4all and complete a scan. ddt4all unexpectedly quits a short moment after I click through to see a screen (screen never shows). 


https://github.com/cedricp/ddt4all/discussions/930#discussioncomment-7490140

**Comment by Furtif** [comment]
fine https://github.com/cedricp/ddt4all/discussions/933

**Comment by Bobbik1** [comment]
what is the problem here:
**Traceback (most recent call last):
  File "C:\Program Files (x86)\ddt4all\parameters.py", line 1040, in <lambda>
    qbutton.clicked.connect(lambda state, btn=qbutton.uniquename: self.buttonClicked(btn))
  File "C:\Program Files (x86)\ddt4all\parameters.py", line 1133, in buttonClicked
    elm_data_stream = ecu_data.setValue(input_value, elm_data_stream, dataitem, ecu_request.ecu_file.endianness)
  File "C:\Program Files (x86)\ddt4all\ecu.py", line 635, in setValue
ValueError: invalid literal for int() with base 2: '0b00b10011'**

Screens loads, but i cant save any config
